src/pycram/process_modules/hsrb_process_modules.py

```diff
@@ -237,45 +36,18 @@ class HSRBNavigationReal(ProcessModule):
     def _execute(self, designator: MoveMotion) -> Any:
         rospy.logdebug(f"Sending goal to giskard to Move the robot")
         # giskard.achieve_cartesian_goal(designator.target, robot_description.base_link, "map")
-        queryPoseNav(designator.target)
-
-
-class HSRBNavigationSemiReal(ProcessModule):
-    """
-    Process module for the real HSRB that sends a cartesian goal to giskard to move the robot base
-    """
-
-    def _execute(self, designator: MoveMotion) -> Any:
-        rospy.logdebug(f"Sending goal to giskard to Move the robot")
-        giskard.achieve_cartesian_goal(designator.target, RobotDescription.current_robot_description.base_link, "map")
+        # todome fix this
```

Comment:
this will be fixed when move/base pr that will come is fixed

Reply:
You can wirte # TODO fix this and then scan with pycharm for all todos, so you dont have to remember that yourself :sunglasses:

Reply:
todome is a self made command for me that im filtering in pycharm so i now I HAVE to do that :D